### ISSUE TITLE
variable_op_scope and variable_scope

### ISSUE THREAD

**Issue by lfz** [tflearn/tflearn#287]
I compiled a tensorflow from source, and I found the following warning when I used batchnorm in tflearn:

`WARNING:tensorflow:tf.variable_op_scope(values, name, default_name) is deprecated, use tf.variable_scope(name, default_name, values)`

It seems that syntax of "variable_scope" changed a lot in the bleeding edge tensorflow.

**Comment by aymericdamien** [comment]
Oh thanks for reporting that, we will have a check on how to handle it.

**Comment by gonzalolc** [comment]
I'm having the same issue.  how can I fix this?
